dspace/config/modules/researcher-profile.cfg

```diff
@@ -0,0 +1,17 @@
+#---------------------------------------------------------------#
+#------------------- PROFILE CONFIGURATIONS --------------------#
+#---------------------------------------------------------------#
+
+# The Entity Type to use for the Researcher Profile. Defaults to "Person" as this is the recommended Entity Type to use
+researcher-profile.entity-type = Person
```

Comment:
@LucaGiamminonni : After submitting my review, I just realized one minor thing we may need to do on the backend.  I think we will need to **DISABLE Researcher Profiles by default in DSpace** (which requires commenting this configuration out), simply because Configurable Entities are also disabled by default. 

Another option would be to add a small test to the backend that proves nothing breaks when the configured `entity-type` **doesn't exist**.   In other words, by default, in a DSpace 7 site, the `Person` entity won't exist, so we need to ensure nothing errors out with that setup.  We could prove this via an IT by simply setting `researcher-profile.entity-type = FakeEntity` and ensuring that nothing errors out (as `FakeEntity` won't exist).